frontends/p4/commonInlining.h

```diff
@@ -169,8 +169,9 @@ class InlineDriver : public Visitor {
  public:
     InlineDriver(
         InlineList* toInline, AbstractInliner<InlineList, InlineWorkList> *inliner) :
-            toInline(toInline), inliner(inliner)
-    { CHECK_NULL(toInline); CHECK_NULL(inliner); setName("InlineDriver"); }
+            toInline(toInline), inliner(inliner) {
+        CHECK_NULL(toInline); CHECK_NULL(inliner);
+        setName((cstring("InlineDriver_") + cstring(inliner->name())).c_str()); }
```

Comment:
This is an unrelated improvement which will cause the various inlining passes to have different names.